### DISCUSSION TITLE
Can not access Props in Emotion

### DISCUSSION THREAD

**Discussion by ibnsina77** [emotion-js/emotion#2390] [Q&A]
Hello Developers!! :)

do you probably know, why I am not able to access props inside emotion styled component?

```
function Grid({ projekti, numberOfProjects }) {

  let calculateGridRows = Math.ceil(numberOfProjects/3); (here it works)

    const GridStyled = styled.div`
      .grid-container {
        display: grid;
        grid-template-columns: repeat(1, auto);

        grid-template-rows: repeat(${(props) => props.numberOfProjects}, auto); (here does not)

      }
```

Thank you all in advance

**Comment by Andarist** [comment]
This code snippet has too many missing pieces for the answer to be given.

Please always try to share a repro case in a runnable form - either by providing a git repository to clone or a codesandbox. OSS maintainers usually can't afford the time to set up the repro, even if exact steps are given.
